src/utils/discord_message.py

```diff
@@ -0,0 +1,37 @@
+from cogs.music.song import Song
+from youtubesearchpython import Comments
+
+CHUNK_SIZE = 10
+TEXT_SIZE = 92
+# Max message size: ((2 + 3 + 3) + (92 * 2) + (2 * 3)) * 10 = 1980
```

Comment:
This can be extracted in a constant and moved where it is actually used (in __queue() in music.py)